stdlib/public/Cxx/CxxDictionary.swift

```diff
@@ -25,6 +25,22 @@ public protocol CxxDictionary<Key, Value> {
 
   /// Do not implement this function manually in Swift.
   func __endUnsafe() -> RawIterator
+
+  func filter(
+    _ isIncluded: (Key, Value) throws -> Bool
+  ) rethrows -> [Key: Value] {
```

Comment:
This won't compile: this function is currently defined within the protocol definition, while it should be declared in an extension for the `CxxDictionary` protocol (see `extension CxxDictionary` in the same file below)

Reply:
only declaration or with it's body ?

Reply:
The entire function declaration with its body should be within an extension for `CxxDictionary`. Swift doesn't require you to declare the function separately in this case:
https://docs.swift.org/swift-book/documentation/the-swift-programming-language/protocols#Protocol-Extensions